mne/viz/utils.py

```diff
@@ -2351,3 +2351,56 @@ def _generate_default_filename(ext=".png"):
     now = datetime.now()
     dt_string = now.strftime("_%Y-%m-%d_%H-%M-%S")
     return "MNE" + dt_string + ext
+
+
+def marching_cubes(image, level):
```

Comment:
What do you think about making `peak_to_volume` and `mne` function? Maybe renamed to `marching_voxels` or something. Where would it go if so?

Reply:
To me marching cubes (subject of this comment) should go in `mne/surface.py`. But if you're talking about some iEEG-specific (probably) function that maps X/Y/Z locations from one MRI volume into another would probably be useful. But let's get this tutorial working and merged, then hash out an API for this second function in another issue / PR / discussion. Let's make it work, then make it pretty.

Reply:
Oops I missed seeing this comment and made a `voxel_neighbors` function in `mne/surface.py` it would be fine to pull out but the tutorial is a lot cleaner this way. Some of the slicing is really better not shown in a tutorial, seems unnecessarily confusing although generalizable to N dimensions.